### DISCUSSION TITLE
Please help to launch OpenDrift graphical interface

### DISCUSSION THREAD

**Discussion by kettlehead1** [OpenDrift/opendrift#1044] [Q&A] [answered]
Hello. Could you send instructions  how to launch the graphical interface after installation

**Comment by knutfrode** [comment]
Hi,
After installation as described here https://opendrift.github.io/install.html  (note the `pip install` at the end)
you can start the GUI by typing `opendrft_gui` from commandline.

Alternatively you may enter the folder `~opendrift/opendrift/scripts/` and type `python opendrift_gui.py`

**Reply by kettlehead1** [reply]
Thanks for the reply but I got an error
opendrft_gui : The name "opendrft_gui" is not recognized as the name of a cmdlet, function, script file, or executable.
Check the spelling of the name and the presence and correctness of the path, then try again.
string:1 character:1
+ opendrft_gui
+ ~~~~~~~~~~~~
      + CategoryInfo : ObjectNotFound: (opendrft_gui:String) [], CommandNotFoundException
      + FullyQualifiedErrorId : CommandNotFoundException

**Reply by kettlehead1** [reply]
I use  miniconda3

**Reply by knutfrode** [reply, marked as answer]
ok, seems you are using a Windows machine. Then you should probably google how to launch an application from a powershell. Might be enough with `. <command>`
